vic/drivers/cesm/bld/vic.buildexe.csh

```diff
@@ -0,0 +1,23 @@
+#! /bin/csh -f
+
+cd $OBJROOT/lnd/obj
+
+set comp = 'unknown'
+if ($COMP_INTERFACE == 'MCT' ) set comp = mct
+if ($COMP_INTERFACE == 'ESMF') set comp = esmf
+
+cat >! Filepath << EOF
+$CASEROOT/SourceMods/src.vic
+$CODEROOT/lnd/vic/vic
+$CODEROOT/lnd/vic/vic/vic_run/src
+$CODEROOT/lnd/vic/vic/vic_run/include
+$CODEROOT/lnd/vic/vic/drivers/shared/src
+$CODEROOT/lnd/vic/vic/drivers/shared/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/src
+$CODEROOT/lnd/vic/vic/drivers/cesm/include
+$CODEROOT/lnd/vic/vic/drivers/cesm/cpl_$comp/
+EOF
+
+set vicdefs = ""
```

Comment:
Okay, I'm going to leave this for now and may ask you to step in if we end up having problems with the build without these flags.  Currently, the only flags we're getting with `icc` are: `icc -c -I... -O1 -fp-model precise   -DLINUX  -DMCT_INTERFACE -DHAVE_MPI -DFORTRANUNDERSCORE -DNO_R16 -DHAVE_F2008_CONTIGUOUS -DLINUX -DCPRINTEL  -DHAVE_SLASHPROC`.